slash/react/src/index.ts

```diff
@@ -12,6 +12,7 @@ export {
   CheckboxModes,
 } from "./Form/Checkbox";
 export { Choice, ChoiceInput } from "./Form/Choice";
+export { Summary } from "./Summary/Summary";
```

Comment:
Peut-être mettre un index dans Summary afin de pouvoir l'appeler en faisant : 
```suggestion
export { Summary } from "./Summary";
```

Reply:
je peux renommer ce fichier en index.tsx, mais rajouter un fichier barrel file pour 1 seul export me semble un peu overkill.
cf: https://dev.to/tassiofront/barrel-files-and-why-you-should-stop-using-them-now-bc4

Je ne suis pas aussi zélé que le monsieur de l'article, mais si je peux éviter des fichiers non nécessaires je préfère :)

Reply:
j'ai rename en index, ca rend bien 👍